### ISSUE TITLE
Enhancement Request: Do a static analysis on a github profile and match it against results from stack overflow.

### ISSUE THREAD

**Issue by colonelpopcorn** [adamisntdead/Overflowing#1]
This is great, but copying isn't a great metric to measure actual use of code from Stack Overflow. It would be a much more useful application if we could compare code in a GitHub repo to results from Stack Overflow. Basically, GitHub knows the code we've written (at least most of it), and Stack Overflow has the code we copied. Just compare the two and see what shakes out!

**Comment by adamisntdead** [comment]
Yeah, I thought of something like that, but there are so many solutions you could have learned off already, and there's so much on Stack Overflow, it would be hard to have an accurate metric, apart from a small personal one such as the one which this repo was created to look at.

Good suggestion though
